2-mandatory/1-create-functions.js

```diff
@@ -44,7 +50,18 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(array) {
```

Comment:
Great job. This solution works well.

If you were to refactor your solution to use a map, how would you do that?

Start practising maps now as you will use them all the time in React.